demos/astro-playground/astro.config.mjs

```diff
@@ -31,6 +31,6 @@ export default defineConfig({
         enabled: false, // tmp deactivate until version after 7.83.0
       },
     }),
-    spotlight(),
+    spotlight({ debug: true }),
```

Comment:
is this on purpose?

Reply:
nope. It probably doesn't really matter, it's a demo/playground app and debug logs were helpful here 😅 

but I'll remove it again

Reply:
oh nvm, 7.97.0 it is (missed that the 7.96.0 release was incomplete)